Mathlib/SetTheory/Ordinal/Arithmetic.lean

```diff
@@ -118,14 +101,14 @@ private theorem add_lt_add_iff_left' (a) {b c : Ordinal} : a + b < a + c ↔ b <
   rw [← not_le, ← not_le, add_le_add_iff_left]
 
 instance add_covariantClass_lt : CovariantClass Ordinal.{u} Ordinal.{u} (· + ·) (· < ·) :=
-  ⟨fun a _b _c => (add_lt_add_iff_left' a).2⟩
+  ⟨fun a _ _ ↦ (add_lt_add_iff_left' a).2⟩
```

Comment:
Just noting that this makes hovering on the proof term a worse experience

Reply:
Fair enough, I had just assumed this was some holdover from Lean 3 porting.